### PR TITLE
Makes get_own_endpoints async

### PR DESCRIPTION
**Pull request by inetic**
[Review on Reviewable](https://reviewable.io/reviews/maidsafe/crust/320)

### CODE REVIEW

**Comment by maidsafe-highfive** [comment]
r? @vinipsmaker

(maidsafe_highfive has picked a reviewer for you, use r? to override)

File: src/service.rs [hunk 1]
```diff
@@ -16,21 +16,23 @@
...
 use bootstrap_handler::BootstrapHandler;
 use config_handler::{Config, read_config_file};
 use getifaddrs::{getifaddrs, filter_loopback};
 use transport;
 use transport::{Endpoint, Port, Message};
+use ip;
+use map_external_port::async_map_external_port;
 
-use map_external_port::map_external_port;
+//use map_external_port::map_external_port;
```
Comment (vinipsmaker) [inline src/service.rs]: should remove commented code?

Comment (inetic) [inline src/service.rs]: ok

Comment (inetic) [inline src/service.rs]: done

**Pull request merged**